django_extractors/django_path.py

```diff
@@ -0,0 +1 @@
+DJANGO_PATH = 'ENV/lib/python3.4/site-packages/django/'
```

Comment:
I'm rusty on how Atom plugins are written, but is it not a concern being hardcoded to a certain Python version here?